CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 1.8.0
```

Comment:
Semantic versioning suggest there should be no breaking changes in minor versions. So that would be `2.0.0`, but I have more plans for that release, so we'll have to hold it off for now.

Also removing EOL rubies also requires removing code that's in place to support these and is a much bigger change.